src/nsupdate/main/templates/main/includes/tabbed_router_configuration.html

```diff
@@ -281,19 +281,28 @@ <h4>inadyn ({% trans "verified with" %} 1.99.11)</h4>
 
 # note: no IPv6 support in inadyn yet.
 </pre>
-        <h4>inadyn ({% trans "verified with" %} 2.1)</h4>
+        <h4>inadyn ({% trans "verified with" %} 2.10.0)</h4>
         <pre># /etc/inadyn.conf
 
 period       = 300
 
-provider ipv4@nsupdate.info {
+# use either ipv4 or ipv6 configuration below
+# IPv4 support
+#provider ipv4@nsupdate.info {
+#    ssl      = true
+#    username = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+#    password = {{ update_secret|default:"&lt;your secret&gt;" }}
+#    hostname = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+#}
+
+# IPv6 support
+provider ipv6@nsupdate.info {
     ssl      = true
     username = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
     password = {{ update_secret|default:"&lt;your secret&gt;" }}
     hostname = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
 }
```

Comment:
so, updating both ipv4 and ipv6 at the same time does not work from 1 config?

Reply:
did you see my question?

Reply:
"it" means updating both at the same time?

if so, can you update your PR so it does that?

Reply:
thanks!